### PR TITLE
Fixed and fully updated(also completed) Russian translation

### PR DESCRIPTION
**Pull request by TheDmitry**
**What:**
Well, as you read in title we updated, fixed n' fully translated GregTechCEu to Russian. Also we just found out some mistakes in English localization.

**Outcome:**
A lot of fixes in ru_ru.lang file and some fixes in eu_us

### CODE REVIEW

**Comment by TheDmitry** [comment]
Special thanks to Fooof, QSTM, Xikaro, Why(Вай), TheDmitry and everyone else who contributed to our translation on weblate.

**Comment by ALongStringOfNumbers** [comment]
Would you mind adding something to the top of the ru_ru lang file that says what CEu version the translations correspond to?

Similar to
https://github.com/GregTechCEu/GregTech/blob/11941fc6347c1576b829983c5a101781977b8db6/src/main/resources/assets/gregtech/lang/zh_cn.lang#L1-L4

**Comment by TheDmitry** [comment]
> Would you mind adding something to the top of the ru_ru lang file that says what CEu version the translations correspond to?
> 
> Similar to
> 
> https://github.com/GregTechCEu/GregTech/blob/11941fc6347c1576b829983c5a101781977b8db6/src/main/resources/assets/gregtech/lang/zh_cn.lang#L1-L4

Done

**Review by Zalgo239 — APPROVED** [review]

File: src/main/resources/assets/gregtech/lang/en_us.lang [hunk 1]
```diff
@@ -4412,7 +4412,8 @@ gregtech.machine.miner.multi.modes=Has Silk Touch and Chunk Aligned Modes.
 gregtech.machine.miner.multi.tooltip=Max Working Area: §b%dx%d§7 chunks.
 gregtech.machine.miner.multi.production=Produces §b3x§7 more crushed ore than a macerator.
 gregtech.machine.miner.multi.description=A multiblock mining machine that covers a large area and produces huge quantity of ore.
-#Small ores lang: gregtech.machine.miner.multi.production=Produces 3x more crushed ore than a macerator. Small ore Fortune §b%d§7.
+#Small ores lang:
+gregtech.machine.miner.multi.production=Produces 3x more crushed ore than a macerator. Small ore Fortune §b%d§7.
```
Comment (ALongStringOfNumbers) [inline src/main/resources/assets/gregtech/lang/en_us.lang:4416]: This lang key should stay commented out, ie one line starting with `#` as it was before, as small ores are not implemented at this time.

**Comment by marisathewitch** [comment]
Revert en_us.lang file cnahges

**Review by ALongStringOfNumbers — APPROVED** [review]

**Pull request merged**